specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/storage.json

```diff
@@ -0,0 +1,1998 @@
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the location of the geo-replicated secondary for the storage account. Only available if the accountType is Standard_GRS or Standard_RAGRS."
+        },
+        "statusOfSecondary": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status indicating whether the secondary location of the storage account is available or unavailable. Only available if the SKU name is Standard_GRS or Standard_RAGRS.",
+          "enum": [
+            "available",
+            "unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "AccountStatus",
+            "modelAsString": false
+          }
+        },
+        "creationTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the creation date and time of the storage account in UTC."
+        },
+        "customDomain": {
+          "$ref": "#/definitions/CustomDomain",
+          "readOnly": true,
+          "description": "Gets the custom domain the user assigned to this storage account."
+        },
+        "secondaryEndpoints": {
+          "$ref": "#/definitions/Endpoints",
+          "readOnly": true,
+          "description": "Gets the URLs that are used to perform a retrieval of a public blob, queue, or table object from the secondary location of the storage account. Only available if the SKU name is Standard_RAGRS."
+        },
+        "encryption": {
+          "$ref": "#/definitions/Encryption",
+          "readOnly": true,
+          "description": "Gets the encryption settings on the account. If unspecified, the account is unencrypted."
+        },
+        "accessTier": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Required for storage accounts where kind = BlobStorage. The access tier used for billing.",
+          "enum": [
+            "Hot",
+            "Cool"
+          ],
+          "x-ms-enum": {
+            "name": "AccessTier",
+            "modelAsString": false
+          }
+        },
+        "azureFilesAadIntegration": {
+          "type": "boolean",
+          "default": false,
+          "x-ms-client-name": "EnableAzureFilesAadIntegration",
+          "description": "Enables Azure Files AAD Integration for SMB if sets to true."
+        },
+        "supportsHttpsTrafficOnly": {
+          "type": "boolean",
+          "default": false,
+          "x-ms-client-name": "EnableHttpsTrafficOnly",
+          "description": "Allows https traffic only to storage service if sets to true."
+        },
+        "networkAcls": {
+          "$ref": "#/definitions/NetworkRuleSet",
+          "x-ms-client-name": "NetworkRuleSet",
+          "description": "Network rule set",
+          "readOnly": true
+        },
+        "isHnsEnabled": {
+          "type": "boolean",
+          "default": false,
```

Comment:
Based on a recent issue found by @williexu in the CLI, please also remove this default value.